mms/decoder.go

```diff
@@ -414,7 +414,7 @@ func (dec *MMSDecoder) Decode(pdu MMSReader) (err error) {
 		case X_MMS_RETRIEVE_TEXT:
 			_, err = dec.ReadString(&reflectedPdu, "RetrieveText")
 		case X_MMS_MMS_VERSION:
-			_, err = dec.ReadShortInteger(&reflectedPdu, "Version")
+			_, err = dec.ReadByte(&reflectedPdu, "Version")
```

Comment:
That is right, however if we do that we need to change the definitions of MMS_MESSAGE_VERSION_1_X in mms.go and take this into account also when encoding the version field.